src/components/ReactTable/ReactTable.tsx

```diff
@@ -1,25 +1,25 @@
 import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
 import { Flex, Table, Tbody, Td, Th, Thead, Tr, useColorModeValue } from '@chakra-ui/react'
 import { useMemo } from 'react'
-import { Column, TableState, useSortBy, useTable } from 'react-table'
+import { Column, Row, TableState, useSortBy, useTable } from 'react-table'
 
-type ReactTableProps = {
-  columns: Column<any>[]
-  data: object[]
+type ReactTableProps<T extends object> = {
```

Comment:
```suggestion
type ReactTableProps<T extends {}> = {
```

try this if possible, `object` is very broad in TS

Reply:
https://www.reddit.com/r/typescript/comments/tq3m4f/the_difference_between_object_and_recordstring/

Reply:
Not sure if this is possible as react-table types are object everywhere but let's give it a try

Reply:
Worked fine

Reply:
Would it still work with more accurate typings then? If any object is accepted, a narrowed down object type should work as well.